src/universalindentgui.cpp

```diff
@@ -8,22 +8,50 @@
  * @license GPL v2+
  */
 #include "universalindentgui.h"
+
+#include "error_types.h"
 #include "prototypes.h"
-#include "uncrustify_version.h"
 #include "unc_ctype.h"
 #include "uncrustify.h"
-#include "error_types.h"
-#include <stdio.h>
+#include "uncrustify_version.h"
+
+#include <vector>
+
+#include <cstdio>
 
 
 using namespace std;
-using namespace uncrustify;
```

Comment:
(Note: https://github.com/uncrustify/uncrustify/pull/1986#discussion_r214538239 — done! :wink: I'd actually forgotten that happened here...)